library/src/main/java/com/novoda/downloadmanager/lib/DatabaseHelper.java

```diff
@@ -24,8 +24,7 @@
 
     private static final String VERSION_ONE_TO_VERSION_TWO_MIGRATION_SCRIPT = "ALTER TABLE "
             + DownloadContract.Batches.BATCHES_TABLE_NAME
-            + " ADD "
-            + DownloadContract.Batches.COLUMN_HAS_STARTED + " BOOLEAN NOT NULL DEFAULT 0;";
+            + ";";
```

Comment:
:-1:  can't just do this I'm afraid, you'll need to create another migration that removes the column